compiler/ballerina-lang/src/main/java/io/ballerina/projects/BalaWriter.java

```diff
@@ -71,9 +72,10 @@
 
     // Set the target as any for default bala.
     protected String target = "any";
-    protected String langSpecVersion = "2020r2";
-    protected String ballerinaVersion = "Ballerina 2.0.0";
-    protected String implemetationVendor = "WSO2";
+    private String langSpecVersion = "2020r2";
```

Comment:
Dont we have to use `RepoUtils.getSpecVersion()` here as well

Reply:
Removed `langSpecVersion` since it's not needed anymore with the new project API changes.